docs/autogenerate/Cachix.adoc

```diff
@@ -1,38 +1,33 @@
 If you're using more than one agent or would like to share
 resulting binaries outside the build farm you'll need a binary cache.
 
-On https://cachix.org[Cachix] you can create a binary cache and
-instruct the agent how to use it:
+On https://cachix.org[Cachix] you can create a binary cache and get the following keys:
+
+signingKeys:: Secret signing key to sign each store path obtained via `cachix generate-keypair` and found in `~/.config/cachix/cachix.dhall`
```

Comment:
They will most likely be generating the key, so it's better to instruct where it's printed out. Those having a backup will either remember they did it or they didn't do it at all.

Reply:
I don't think we can tell them to run the command as part of this process. They will need to follow the cachix instructions anyway, so duplicating part of those instructions here only serves as a reference, but in the worst case confuses people because they'll try to run it immediately.

If they run it immediately, without the cachix.org instructions, it's not obvious what to do with the cache name.
If they run it after following the instructions on cachix.org, they will be faced with an error that a key has already been generated.

By referencing cachix.org, I intend to sidestep the problem of invoking that command at the wrong time.

Of course I am open to another phrasing that addresses the limitations of `generate-keypair`.
Perhaps we should tell them to follow the cachix.org steps first, instead of at the end of the sentence.

Reply:
That's a good point, we should tell them to complete binary cache creation and continue the process here.